src/components/FollowersList/FollowersList.js

```diff
@@ -0,0 +1,30 @@
+import FollowerDetails from '../FollowerDetails/FollowerDetails'
+import './FollowersList.css'
+
+const FollowersList = ({ followers, visible, setVisible}) => {
+
+  const toggleList = () => {
+    (!visible) ? setVisible(true) : setVisible(false)
+  }
+
+  const renderList = () => {
+    return (
+      <div>
+      {followers.map((follower, index) => <FollowerDetails key={index} id={follower.friendId} followersVisible={setVisible}/>)}
+      </div>
+    )
+  }
+
+  return (
+    <section className={!visible ? "follow-list hidden" : "follow-list"}>
+      {
+        !!visible && <div>
+        <button onClick={toggleList}>Close</button>
```

Comment:
These are the lines of code you're looking for! I appreciate that they're moved into a new component.